src/SparkFun_u-blox_GNSS_Arduino_Library.h

```diff
@@ -560,6 +562,8 @@ class SFE_UBLOX_GNSS
 	boolean begin(TwoWire &wirePort = Wire, uint8_t deviceAddress = 0x42); //Returns true if module is detected
 	//serialPort needs to be perviously initialized to correct baud rate
 	boolean begin(Stream &serialPort); //Returns true if module is detected
+	//SPI - supply instance of SPIClass, slave select pin and SPI speed (in Hz)
```

Comment:
Please change ```slave``` to ```chip```: https://www.oshwa.org/2020/06/29/a-resolution-to-redefine-spi-pin-names/

Reply:
Done. Commit pending